thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/util/CacheUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.util;
+
+import com.couchbase.client.java.document.json.JsonObject;
+import java.util.zip.CRC32;
+import org.apache.pinot.thirdeye.detection.cache.TimeSeriesDataPoint;
+
+
+public class CacheUtils {
+
+  public static String hashMetricUrn(String metricUrn) {
+    CRC32 c = new CRC32();
+    c.update(metricUrn.getBytes());
+    return String.valueOf(c.getValue());
+  }
+
+  public static JsonObject buildDocumentStructure(TimeSeriesDataPoint point) {
+    JsonObject body = JsonObject.create()
+        .put("time", point.getTimestamp())
+        .put("metricId", point.getMetricId())
+        .put(point.getMetricUrnHash(),
+            (point.getDataValue() == null || point.getDataValue().equals("null")) ? "0" : point.getDataValue());
```

Comment:
What about "NULL"? how do we handle null value here?

Reply:
fixed this and added it to .getDataValue() method. Now, we check whether dataValue.toLowerCase() is equal to "null" rather than direct comparison to "null".